src/kirby/components/card/card.component.tns.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, Input, OnInit, ViewChild, ElementRef, AfterViewInit } from '@angular/core';
```

Comment:
Ville du ikke flytte det {N} specifikke til en service, så vi kun har én component-klasse/-fil?

Reply:
Jeg er ikke enig her. Jeg bruger @ViewChild i komponenten for at få outer og inner views, og det skal jeg blive ved med for at det virker, så der skal alligevel være en card.component.tns.ts for at det virker. Jeg syntes at når man har disse afhængiheder, så giver det mening med seperat tns til komponenter, og så kan man lave en fælles service til det fælles logik/kode i stedet for, altså vende det om.

Reply:
Hmmmm, ok, men hvis vi kun bruger det her, så kunne vi vel bruge `(loaded)="applyShadow(outerView, innerView)"`?